code/modules/mob/living/silicon/decoy/decoy.dm

```diff
@@ -4,9 +4,53 @@
 	icon_state = "ai"
 	anchored = 1 // -- TLE
 	canmove = 0
+	a_intent = "harm" // This is apparently the only thing that stops other mobs walking through them as if they were thin air.
 
 /mob/living/silicon/decoy/New()
 	src.icon = 'icons/mob/AI.dmi'
 	src.icon_state = "ai"
 	src.anchored = 1
-	src.canmove = 0
\ No newline at end of file
+	src.canmove = 0
+
+/mob/living/silicon/decoy/attackby(var/obj/item/W, var/mob/user, params)
+	if(istype(W, /obj/item/aicard))
+		user.visible_message("<span class='notice'>[user] cannot find an intellicard slot on [src].</span>")
+	else
+		return ..(W, user, params)

```

Comment:
the vars on all `attackby` should be the same, almost without exception, this should be `return ..()`